src/core/importType.js

```diff
@@ -97,5 +102,5 @@ export function isScopedModule(name) {
 }
 
 export default function resolveImportType(name, context) {
-  return typeTest(name, context.settings, resolve(name, context))
+  return typeTest(name, context.settings, resolve(name, context), context)
```

Comment:
```suggestion
  return typeTest(name, context, resolve(name, context))
```
that way typeTest can read the settings off of the context object instead.

Reply:
done